index.html

```diff
@@ -1,30 +1,125 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="style.css">
   <title>Document</title>
 </head>
-<body>
 
-  <h1>Awesome Books</h1>
+<body>
+  <div class="container">
 
-  <div class="books-container">
+    <div class="navigation">
+      <div class="nav-name">
+        <p>Awesome Books</p>
+      </div>
+      <div>
+        <ul class="nav-list">
+          <li class="active list-item"><a href="#">List</a> </li>
+          <li class="add-item"><a href="#">Add New</a></li>
+          <li class="contact-item"><a href="#">Contact</a></li>
+        </ul>
+      </div>
+<<<<<<< HEAD
+      
+    </div>
+    <div class="date">
+      <script>document.write(new Date())</script>
+=======
```

Comment:
- [ ] Great job so far. However, there're few things to work on before you could move on to the next milestone. Your website must look almost exactly like the wireframe given. There some unwanted elements on your page when it was opened. Kindly take a look at below screen recording and the wireframe and remove all the unwanted elements from your page.

###  Your Website
[app.webm](https://github.com/Prisca-tech/AWESOME_BOOKS/assets/26597432/f8bcb8df-5ed7-4881-a9bc-4e082f12ed1d)

### Wireframe
![wireframe](https://github.com/Prisca-tech/AWESOME_BOOKS/assets/26597432/7b87207f-7650-4a7b-b52d-cded03833519)